SetAPI.spec

```diff
@@ -628,7 +628,26 @@ module SetAPI {
     funcdef create_sample_set(CreateRNASeqSampleSetParams params)
                       returns(CreateRNASeqSampleSetResult) authentication required;
 
+    typedef tuple<string column, boolean ascending> column_sorting;
 
+    typedef structure {
```

Comment:
Some commentary here would be helpful. Are there limits on how many you can request (the "limit" value, I'm guessing has some max)? What column strings are allowed? Is there an expected format for the query string? And so on.